x-pack/plugins/siem/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": "Elastic",
+  "name": "siem",
+  "version": "8.0.0",
+  "private": true,
+  "license": "Elastic-License",
+  "scripts": {
+    "extract-mitre-attacks": "node scripts/extract_tactics_techniques_mitre.js & node ../../../scripts/eslint ../../legacy/plugins/siem/public/pages/detection_engine/mitre/mitre_tactics_techniques.ts --fix",
```

Comment:
When running `yarn extract-mitre-attacks` I'm seeing this diff with the generated file. Seems there's a difference in the lint config?

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/79516793-a33d8380-8009-11ea-9c49-beb024cf25e0.png" />
</p>

Reply:
Yeah, I noticed that as well but forgot to circle back. I'll verify that it's working correctly on master and then figure out what's changed.

Reply:
It looks like the eslint forget to do its jobs maybe the path change
`node scripts/extract_tactics_techniques_mitre.js & node ../../../../scripts/eslint ./public/pages/detection_engine/mitre/mitre_tactics_techniques.ts --fix`

Reply:
I figured out the fix in #64010 👀